profiles/kentik_snmp/_general/net-snmp.yml

```diff
@@ -22,6 +22,7 @@ matches:
   "^uap-ac-inwall": unifi-access-point.yml
   "root@sensey": opnsense-protectli.yml
   "poweriq": power-iq.yml
+  "#1 SMP": sonicwall-sma.yml
```

Comment:
This should be lower case though. `# Regex is run on _lower case_ version of the sysDesc string.`

We convert the sysdesc string to lower case before applying any of the regex matches.

Reply:
got it; ty